### PR TITLE
IDMP-398 - Add concept for Marketing Status (SPOR RMS)

### PR DESCRIPTION
**Pull request by ElisaKendall**
Revised approach to controlled vocabularies in SPOR RMS and added a controlled vocab for marketing status for SPOR

Signed-off-by: Elisa Kendall <ekendall@thematix.com>

### CODE REVIEW

File: ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf [hunk 1]
```diff
@@ -160,59 +190,143 @@
...
-		<rdfs:label xml:lang="fr-FR">Usage humain</rdfs:label>
-		<rdfs:label xml:lang="it">uso umano</rdfs:label>
-		<skos:definition>substance domain classifier for a substance that is authorized for use in human populations</skos:definition>
+		<rdfs:label>100000000012</rdfs:label>
 		<cmns-av:abbreviation xml:lang="de">H</cmns-av:abbreviation>
 		<cmns-av:abbreviation xml:lang="en">H</cmns-av:abbreviation>
 		<cmns-av:abbreviation xml:lang="fr-FR">H</cmns-av:abbreviation>
 		<cmns-av:abbreviation xml:lang="it">H</cmns-av:abbreviation>
 		<cmns-av:abbreviation xml:lang="sv">H</cmns-av:abbreviation>
+		<cmns-av:synonym xml:lang="de">Human</cmns-av:synonym>
```
Comment (tw-osthus) [inline ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf:199]: This seems inconsistent.
If the label is 100000000012 then all textual names are synonyms of it and also "Human". 
We are struck here with DatatypeProperties, and you would need some kind of skos-xl if you want to express this. In German you have a preferred  textual name "Anwendung am Menschen" which means "human use" and a broader alternative textual name (synonym) "Human" translating to "human related" in English, but you cannot express it on a literal.
If hasTextualName has cardinality 1 per language then we need to add hasAlternateTextualName as predicate to add "Human" or we should drop it.
Or add all textual names as synonyms as well.

Comment (ElisaKendall) [reply ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf:199]: @tw-osthus The way I've reframed this corresponds with what is in the SPOR reference list. The label in SPOR is this numeric identifier, which is also the last part of the URI. The term name is the primary field that everything maps to, which is why I thought use of the textual name property was the right thing to do - it is the field used in their UI, and what appears in the data as the value for a given element in a controlled vocabulary. They do this consistently in all of their controlled vocabularies in fact. 

I can create specific properties for each of the fields, which are data properties, if that would be clearer than the approach I've taken using the annotations for the other properties, but the main things are the identifier and term name for any element in these controlled vocabularies - the rest are not as essential.

**Review by tw-osthus — APPROVED** [review]

File: ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf [hunk 2]
```diff
@@ -158,60 +213,206 @@
...
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="https://spor.ema.europa.eu/rmswi/#/lists/200000004983/terms/200000004985">
+		<rdf:type rdf:resource="&idmp-spor;SPOR-DataClassification"/>
+		<rdfs:label>200000004985</rdfs:label>
+		<idmp-spor:hasSPORTermDescription xml:lang="en">data classification for a substance or product that is considered to be public from an accessibility perspective</idmp-spor:hasSPORTermDescription>
+		<idmp-spor:hasSPORTermName xml:lang="en">Public</idmp-spor:hasSPORTermName>
+		<cmns-cls:isClassifiedBy rdf:resource="https://spor.ema.europa.eu/rmswi/#/lists/100000000004/terms/100000000014"/>
+		<cmns-cls:isClassifiedBy rdf:resource="https://spor.ema.europa.eu/rmswi/#/lists/200000004983/terms/200000004985"/>
+		<cmns-dsg:isDefinedIn rdf:resource="&idmp-spor;ReferentialsManagementServicesClassificationScheme"/>
```
Comment (mereolog) [inline ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf:406]: Is 200000004985 classified by itself?

File: ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf [hunk 3]
```diff
@@ -158,60 +213,206 @@
...
+		<idmp-spor:hasSPORTermDescription xml:lang="en">domain classifier for a substance or product that is authorized for use in human and veterinary populations</idmp-spor:hasSPORTermDescription>
+		<idmp-spor:hasSPORTermName xml:lang="de">Anwendung an Mensch und Tier</idmp-spor:hasSPORTermName>
+		<idmp-spor:hasSPORTermName xml:lang="en">Human and Veterinary use</idmp-spor:hasSPORTermName>
+		<idmp-spor:hasSPORTermName xml:lang="no">Human og veterinær bruk</idmp-spor:hasSPORTermName>
+		<idmp-spor:hasSPORTermName xml:lang="sv">Humant och veterinärmedicinskt bruk</idmp-spor:hasSPORTermName>
+		<idmp-spor:hasSPORTermName xml:lang="cs">Humánní a veterinární použití</idmp-spor:hasSPORTermName>
+		<idmp-spor:hasSPORTermName xml:lang="lv">Lietošanai cilvēkiem un dzīvniekiem</idmp-spor:hasSPORTermName>
+		<idmp-spor:hasSPORTermName xml:lang="fr-FR">Usage humain et vétérinaire</idmp-spor:hasSPORTermName>
+		<idmp-spor:hasSPORTermName xml:lang="it">uso umano e veterinario</idmp-spor:hasSPORTermName>
+		<cmns-cls:isClassifiedBy rdf:resource="https://spor.ema.europa.eu/rmswi/#/lists/100000000004/terms/100000000014"/>
```
Comment (mereolog) [inline ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf:306]: Is 100000000014 classified by itself?

File: ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf [hunk 4]
```diff
@@ -158,60 +213,206 @@
...
+		<cmns-cls:isClassifiedBy rdf:resource="https://spor.ema.europa.eu/rmswi/#/lists/200000004983/terms/200000004985"/>
```
Comment (mereolog) [inline ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf:307]: 100000000014 is classified by 200000004985 and 200000004985 is classified by 100000000014.
Is this right?

File: ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf [hunk 5]
```diff
@@ -158,60 +213,206 @@
 		<cmns-prd:isProvidedBy rdf:resource="&idmp-eura;EuropeanMedicinesAgencyRegistrationAuthority"/>
 	</owl:NamedIndividual>
 	
+	<owl:DatatypeProperty rdf:about="&idmp-spor;hasSPOROtherName">
+		<rdfs:subPropertyOf rdf:resource="&lcc-lr;hasTextualName"/>
+		<rdfs:label>has SPOR other name</rdfs:label>
+		<skos:definition>provides a language-tagged string value indicating an alternate name of a given term</skos:definition>
```
Comment (mereolog) [inline ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf:219]: I would prefer 'an alternate name of a given entity' or 'an alternate name of a given resource'.

File: ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf [hunk 6]
```diff
@@ -158,60 +213,206 @@
...
+	</owl:DatatypeProperty>
+	
+	<owl:DatatypeProperty rdf:about="&idmp-spor;hasSPORShortName">
+		<rdfs:subPropertyOf rdf:resource="&lcc-lr;hasTextualName"/>
+		<rdfs:label>has SPOR short name</rdfs:label>
+		<skos:definition>provides a language-tagged string value indicating an abbreviated name of a given term</skos:definition>
```
Comment (mereolog) [inline ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf:225]: I would prefer 'an abbreviated name of a given entity' or 'an abbreviated name of a given resource'.

File: ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf (hunk 2 above)
Comment (ElisaKendall) [reply ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf:406]: Yes - it's both a meta classifier and classifier - a bit odd, but that's the way the data has been implemented in SPOR.

File: ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf (hunk 4 above)
Comment (ElisaKendall) [reply ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf:307]: Yup. See above.

File: ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf (hunk 5 above)
Comment (ElisaKendall) [reply ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf:219]: In the SPOR online repo they are called terms.  This is a SPOR specific usage - though not unlike other things we've seen with controlled vocabularies in other domains.

File: ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf (hunk 6 above)
Comment (ElisaKendall) [reply ISO/EuropeanJurisdiction/SubstancesProductsOrganisationsReferentials.rdf:225]: I agree under normal circumstances, but this corresponds to precisely what is in the SPOR repo.

**Review by mereolog — APPROVED** [review]

**Pull request merged**